libs/ledger-live-common/src/naming-service/logic.ts

```diff
@@ -11,13 +12,12 @@ export function isOutdated(resource: NamingServiceStatus): boolean {
       return now - resource.updatedAt > 60 * 1000; // 1 minute
     }
     case "error": {
-      return now - resource.updatedAt > 1 * 1000; // 1 second
+      return now - resource.updatedAt > 30 * 1000; // 30 seconds
```

Comment:
Not sure 30seconds cannot be an issue here in case of backend failure.
E.g. The backend is under heavy load or simply being updated for a sec so it's not responding, having to wait 30s to redo the call even if the user is changing the address might be an issue I think 🤔

Reply:
Yeah good point I was more like, if someone got his address not found we doesn't need to refresh this, this quickly.
I can probably differ error and not found so we can modulate more easily this type of "error"